### DISCUSSION TITLE
How to count mock servers when it gets hits

### DISCUSSION THREAD

**Discussion by selcuktemizsoy** [pactumjs/pactum#266] [Q&A] [answered]
Hello, I want to count how many hits received by my mock server during my tests. Is there a way to get this information? Also I want to modify it during tests? 

My use case, I have a mocked endpoint and my application has retry functionality to get success response, I just wanted to make sure that my mock called retry times. Also after getting retry count I want to make it zero without restarting my mock server.

**Comment by ASaiAnudeep** [comment, marked as answer]
`expects.callCount` property in [interaction](https://pactumjs.github.io/api/mock/interaction.html) would help to assert on the call count of it. I don't think there is a way to modify the call counts. But we can [remove](https://pactumjs.github.io/api/mock/removeInteraction.html) the interaction and add it again.

**Reply by selcuktemizsoy** [reply]
I solved my problem with adding handler and removing handler. during test I call callCount and verify my assertion. Thanks it helped.
